docs/user-guide/admin-cli.md

```diff
@@ -653,17 +698,112 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
+
+```
+[root@localhost:~] vmdkops_admin config init -h
+usage: vmdkops_admin.py config init [-h] [--local] [--force]
+                                    [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Allows local (SingleNode) Init
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
+                        Config DB will be placed on a shared datastore
+```
+
+Example:
+
+```
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: n/a
+DB_Mode: NotConfigured (no local DB or symlink)  <===== NOT CONFIGURED
+Pid: 5979199
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+
+[root@localhost:~] vmdkops_admin config init --local
+Creating new DB at /etc/vmware/vmdkops/auth-db
+Restarting the vmdkops service to pick up new configuration
+Stopping vmdkops-opsd with PID=5979199
+vmdkops-opsd is not running
+Starting vmdkops-opsd
+vmdkops-opsd is running pid=5979684
+
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: /etc/vmware/vmdkops/auth-db
+DB_Mode: SingleNode (local DB exists)    <==== LOCAL configuration
+Pid: 5979684
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+```
+
+
+#### Remove
+
+Allows to remove configuration. Either `--local` or `--datastore` flag needs to be passed.
+Also to make sure it is not typed accidentally, the command requires `--force` flag to actually execute the request
+
+```
+[root@localhost:~] vmdkops_admin config remove -h
+usage: vmdkops_admin.py config remove [-h] [--local] [--no-backup] [--force]
+                                      [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Removes only local link or local DB
+  --no-backup           Do not create DB backup before removing
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
+                        Remove Config DB from a specific datastore
+```
+
+
+#### Status
+
+To get config DB  status, use  `/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py status` command.
+
+#### Move
+
+Allows to relocate config DB between datastores. [Not implemented yet]
 ## Status
 
 Show config and run-time information about the service.
 
 ```bash
 [root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py status
- Version: 1.0.0-0.0.1
- Status: Running
- Pid: 161104
- Port: 1019
- LogConfigFile: /etc/vmware/vmdkops/log_config.json
- LogFile: /var/log/vmware/vmdk_ops.log
- LogLevel: INFO
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: /etc/vmware/vmdkops/auth-db
```

Comment:
I want to show that one of them is a link (may be a link). I'll update this to match your suggestion closer though.